model/device.go

```diff
@@ -31,6 +35,7 @@ type DeviceAttribute struct {
 	Name        string      `json:"name" bson:",omitempty"`
 	Description *string     `json:"description,omitempty" bson:",omitempty"`
 	Value       interface{} `json:"value" bson:",omitempty"`
+	Scope       string      `json:"scope" bson:",omitempty"`
```

Comment:
Is it really necessary to have a dedicated scope attribute? Wouldn't it suffice to give the name attribute a path-like structure and interpret each separator as a scope separator?
This way we would also achieve nested scopes too.

Reply:
@alfrunes yes, it is needed because we don't want to touch API v1, thus inventory scope should be transparent to API consumers. A new (optional, with default) attribute is fine, changing the name with a prefix is not.

Reply:
I'd suggest not having a default at all - the value is simply is what is specified. If one or more separators are specified the name is parsed into scope(s). 
An exception can be added to the parser, setting a default `inventory` scope, but I can't see why we would want this.
Well... I can see the complexity is about the same as already done here (if not more), so ignore the over-complicating part in my review comment ^ :)

Reply:
@alfrunes we need the default because we are modifying the API v1, which is already consumed in the real world and we cannot break back-compatibility. @mchalski what's your opinion on this?

Reply:
the last thing Mender needs is nested scopes ;) I hope no acceptance criteria mentions this...?

I prefer this implementation as is. clean name, clean scope, it translates verbatim to api responses, minimal translation... mind you we're already doing funny stuff with the attribute struct (prefixed keys, dropped at marshalling).

so short story - I prefer it as is.